src/module-1/cards-list-v1/styles/style.css

```diff
@@ -0,0 +1,149 @@
+* {box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    }
+    
+    body {
+        background-color: #ccc;
+        padding: 40px;
+    }
+
+    .product-list {
+        display: flex;
+        flex-wrap: wrap;
```

Comment:
try to place cards by justify-content and try to add some padding to product-list and remove margin-left from list item